app/src/androidTest/java/com/yacgroup/yacguide/database/Common.kt

```diff
@@ -56,6 +64,18 @@ class TestDB {
             Sector(3, 3f, "Sector3", 2)
         )
 
+        val ROCKS = listOf(
+            Rock(1, 1f, 'G', ' ', "Rock1", 0f, 0f, 0, 1),
+            Rock(2, 2f, 'G', ' ', "Rock2", 0f, 0f, 0, 1),
+            Rock(3, 3f, 'G', ' ', "Rock3", 0f, 0f, 0, 2)
+        )
+
+        val ROCK_COMMENTS = listOf(
+            RockComment(1, 1, "", 1),
+            RockComment(2, 3, "", 1),
+            RockComment(3, 4, "", 2)
```

Comment:
You described the database structure above with different rock comments but you don't test them?

Reply:
What do you mean? This definition matches exactly the database structure above.

Reply:
Maybe I don't understand the code correctly but I would expect somehow the different rock comments e.g. `RockComment1` here?

Reply:
Oh, you mean the name? -> `RockComment(1, 1, "RockComment1", 1)` ? That's not important for the test but I can add it.